input/fsh/omrs-person.fsh

```diff
@@ -0,0 +1,58 @@
+Profile: OMRSPerson
+Parent: Person
+Id: omrs-person
+Title: "OpenMRS Person"
+Description: "A FHIR Person as understood by OpenMRS"
+* identifier 0..*
+* name 1..* MS
+* telecom 0..* MS
+* gender 1..1 MS
+* birthDate 1..1 MS
+* address 0..* MS
+* active 1..1 MS
+* active = true
+* photo 0..0
+* managingOrganization 0..0
+* link 0..0
+
+//OpenMRS Person Example
+
+Instance: example-openmrs-person
+InstanceOf: OMRSPerson 
+Usage: #example
+Title: "Openmrs Person"
+Description: "Example OMRS Person Resource"
+* identifier.id = "53fbd0ef-ae10-46a0-9d0a-387917a66d6e"
+* identifier.use = #official
+* identifier.type.text = "OpenMRS ID"
+* identifier.value = "1000Y"
+* name.id = "915c9146-8bdb-4fad-bc51-4af10d041c86"
+* name.family = "John"
+* name.given = "Smith"
+* gender = #male
+* birthDate = "2001-09-08"
+* address.extension.url = "http://fhir.openmrs.org/ext/address"
+* address.use = #home
+* address.country = "Uganda"
+* address.city = "Ntungamo"
+* address.district = "Ntungamo"
+* active = true
+
+/** mapping FHIR HL7 vs OMRS*/
+
+Mapping: PersonMapping
+Source: OMRSPerson
+Target: "http://hl7.org/fhir/3.0/StructureDefinition/Person"
+Id: person-mapping
+Title: "FHIR HL7 vs OMRS "
+Description: "Person is a FHIR resource that is used to represent demographics and administrative information about a person independent of a specific health-related context. FHIR Persons are mapped to OpenMRS Person objects."
```

Comment:
Hello @abertnamanya  , you are missing one attribute of the name to be added into mapping. Nice work man.

Reply:
Oh yeah I did forget to add the name, Am waiting on @pmanko to merge the person resource PR then I could pull the changes into the person-ex since the branch was created from the person-prof branch